xesmf/tests/test_frontend.py

```diff
@@ -170,3 +170,20 @@ def test_regrid_with_1d_grid():
 
     # clean-up
     regridder.clean_weight_file()
+
+
+def test_build_regridder_with_masks():
+    ds_in['mask'] = xr.DataArray(
+        np.random.randint(2, size=ds_in['data'].shape),
+        dims=('y', 'x'))
+    print(ds_in)
+    # 'patch' is too slow to test
+    for method in ['bilinear', 'conservative', 'nearest_s2d', 'nearest_d2s']:
```

Comment:
Shouldn't `conservative_normed` be exercised ?

Reply:
Yep I agree. In fact, all conservative methods must be tested numerically, especially with masking.